meta/Meta.h

```diff
@@ -459,10 +459,6 @@ namespace saimeta
                     _Out_ uint64_t *counters,
                     _In_ sai_stats_mode_t mode);
 
-            sai_status_t meta_validate_query_stats_capability(
```

Comment:
Why this is removed ?

Reply:
I have explained this for your comment in the definition of the function.